tests/cases/pr557_namespacedJsx.tsx

```diff
@@ -0,0 +1,9 @@
+const valid1 = <standard-tag standard-attribute="foo" />;
```

Comment:
Add comment ```// @onlyOwnGrammar - As this has jsx``` at the top to avoid scoping this for TypeScript without react support.